New Feature added-To Delete a member's record

animeshseemendra opened the pull request:
* A new feature added if admin has to delete the record of a member who is no longer the member of the organisation.

dhruvhacks committed:
Update README

dhruvhacks committed:
To help create Database

dhruvhacks committed:
Main cpp file

dhruvhacks committed:
Function's Definations

dhruvhacks committed:
Blank database

dhruvhacks committed:
Issues Cleared

dhruvhacks committed:
Features Updated

dhruvhacks committed:
Update README

dhruvhacks committed:
Update README

dhruvhacks committed:
Feature added

dhruvhacks committed:
Iteration extended

dhruvhacks committed:
Update README

dhruvhacks committed:
Debugged

dhruvhacks committed:
Iterations increased

dhruvhacks committed:
Bugs Fixed

dhruvhacks committed:
Iterations Increased

dhruvhacks committed:
New Edit Feature Added

dhruvhacks committed:
Iterations Added

dhruvhacks committed:
New Edit Feature Added

dhruvhacks committed:
Merge pull request #1 from ncs-jss/dhruv

Dhruv

animeshseemendra committed:
Update main.cpp

animeshseemendra committed:
Update attendance.class

animeshseemendra committed:
Merge pull request #2 from animeshseemendra/animeshseemendra-patch-2

Update attendance.class

animeshseemendra committed:
Merge pull request #1 from animeshseemendra/animeshseemendra-patch-1

Update main.cpp

dhruvhacks reviewed (changes requested):
Compilation errors-
1. in attendance.class, line 291, '(' unbalanced.
2. in main.cpp, case '7's '{' not closed, unplaced after break statement hence unbalancing '{ }'

dhruvhacks reviewed (changes requested):
logical error-
you considered data till specific member to be deleted is reached. Members after this required data are left in old file and get deleted!! major error.

dhruvhacks reviewed (commented):
Nice attempt. Keep it up. :)

animeshseemendra committed:
Update attendance.class

animeshseemendra committed:
Update attendance.class

animeshseemendra committed:
Update main.cpp

animeshseemendra commented:
Thank you for your review.
Sorry for the mistakes , I have updated it with the required changes.

dhruvhacks reviewed (changes requested):
Update the serial numbers of members after deletion. It's missing.
Secondly, display option 7 as "Delete specific member" in menu.

animeshseemendra committed:
Update attendance.class

animeshseemendra committed:
Update attendance.class

animeshseemendra commented:
Changes done
Updated serial numbers of members.
7 option is displayed as "Delete specific member"

dhruvhacks reviewed (changes requested):
last member's data get's repeated with unknown serial number after deletion of any member.

animeshseemendra committed:
Update attendance.class

animeshseemendra committed:
Merge pull request #3 from animeshseemendra/animeshseemendra-patch-3

Update attendance.class

animeshseemendra commented:
I added a feature to delete a member's record from database, but as the program was not running on my computer earlier so I was getting lots of reviews about the errors. Luckily now it runs, and I have removed all the errors that was coming. If still any error comes please notify.

RishabhJain2018 commented:
@animeshseemendra Modify your commit message according to the feature you have added or write about it in a comment.

dhruvhacks reviewed (changes requested):
Last entry in attendance record repeats itself with false serial number.

dhruvhacks commented:
@animeshseemendra 
Old version of this module is shifted to branch ```v1.0```. Master branch of this repository is now being prepared for v2.0. So do change the PR merge request to ```v1.0``` branch instead of ```master```.
